File: backend/src/main/java/org/globe42/CrypticApplication.java
package org.globe42;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.ConfigurableApplicationContext;

@SpringBootApplication
public class CrypticApplication {

	public static void main(String[] args) {
		ConfigurableApplicationContext run = SpringApplication.run(CrypticApplication.class, args);
	}
}

File: backend/src/main/kotlin/org/globe42/CrypticApplication.kt
package org.globe42

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class CrypticApplication {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            runApplication<CrypticApplication>(*args)
        }
    }
}
